lib/v2/issuehunt/maintainer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/funded/:repo': ['running-grass'],
```

Comment:
```suggestion
    '/funded/:username/:repo': ['running-grass'],
```
The path here should follow router.js and `path` in docs